crates/topos-tce-synchronizer/src/checkpoints_collector/tests.rs

```diff
@@ -42,9 +53,73 @@ fn encode() {
     assert_eq!(y, req);
 }
 
-#[test]
-fn sync_same_certificate_twice() {
-    //
+#[rstest]
+#[test_log::test(tokio::test)]
+#[timeout(Duration::from_secs(10))]
+async fn network_test() {
```

Comment:
I'd be more explicit in the likes of `check_fetch_certificates_with_synchronizer` (or something else)